frontend/amundsen_application/static/js/features/ColumnList/index.tsx

```diff
@@ -309,6 +311,18 @@ const ColumnList: React.FC<ColumnListProps> = ({
   });
 
   let formattedColumns: ReusableTableColumn[] = [
+    {
+      title: '',
+      field: 'stats',
+      width: 24,
```

Comment:
Let's make this magic number into a constant.
Maybe `const STATS_COLUMN_WIDTH = 24;`

Reply:
done